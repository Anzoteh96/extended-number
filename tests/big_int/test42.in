982740489246509269904820963935774452877798038946548199913221018995691066440313098103823996213232600719005287475888185270097563
646483090091679648071730349197470563054707647839320519138950558637416544978530656026629066141959065830235590171011303305230617
