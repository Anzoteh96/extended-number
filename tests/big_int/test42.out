Enter two numbers n and m:
n is 982740489246509269904820963935774452877798038946548199913221018995691066440313098103823996213232600719005287475888185270097563
m is 646483090091679648071730349197470563054707647839320519138950558637416544978530656026629066141959065830235590171011303305230617
n + m is 1629223579338188917976551313133245015932505686785868719052171577633107611418843754130453062355191666549240877646899488575328180
n - m is 336257399154829621833090614738303889823090391107227680774270460358274521461782442077194930071273534888769697304876881964866946
n * m is 635325108246292386721672736219734184026325821502926986325782358956742035925402580138122202842399348105817440441173097302657159492991346119169271373567223539941489145861116000057352056526195083330561288383761032326100131125201742739684598756779804686371
n / m is 1
n % m is 336257399154829621833090614738303889823090391107227680774270460358274521461782442077194930071273534888769697304876881964866946
